bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/dto/TripDetail.java

```diff
@@ -74,7 +75,9 @@ public State getEndPosition() {
     }
 
     public long getDurationInMinutes() {
-        return Duration.between(startTime, endTime).toMinutes();
+        Temporal start = startTime;
+        Temporal end = endTime;
+        return (start == null || end == null) ? 0 : Duration.between(start, end).toMinutes();
```

Comment:
I hope this doesn't cause problems...

Reply:
What would you suggest as an alternative?

Reply:
Change the return type of `getDurationInMinutes` to `OptionalLong` or `Optional<Duration>` and then adjust the code in `VehicleHandler` accordingly. Using an Undef state should suffice for the situation where it is missing.